Infrastructure.md

```diff
@@ -0,0 +1,58 @@
+# Infrastructure Setup
+
+The provisioning system depends on a central master, hosted at http://classroom.puppet.com
+
+This master will be configured using classes in the `puppetlabs/bootstrap`
+module and a control repository.
+
+## Classification
+
+* The `classroom.puppet.com` provisioner should be classified with `bootstrap::role::provisioner`.
+* All other nodes should be classified with `bootstrap::role::training` (or as appropriate)
+* The PE Agent and PE MCollective node groups should be configured to match only `classroom.puppet.com`.
+
+## Hiera
+
+Hiera should contain the private SSH key of a GitHub user with read-only access to
+the `puppet-training/courseware` repository. This should be encrypted with `hiera-eyaml`.
+It should also contain the pre-shared key used by instructors for bootstrapping.
+
+``` yaml
+---
+bootstrap::courseware::key: >
```

Comment:
yeah, the secrets should be stored in Hiera, either in a private control-repo or in a separate private repo.